### PR TITLE
Feature/svg from absolute paths

### PR DESCRIPTION
**Pull request by gionatan-lombardi**
Currently, if you try to import `svg` files from a node_module package, the build process will skip it and treat it as an external dependency:
```sh
'my-svg-library/alert.svg' is imported by ..., but could not be resolved – treating it as an external dependency
```
With this PR, thanks to the `resolveId` rollup hook, the consumer will be able to do this.

The only important thing to take in mind is that the **rollup build process needs to be started from the project root directory** (e.g. via npm scripts). This is because the plugin will search the `node_modules` directory, via the [`process.cwd()`](https://github.com/e-e-e/rollup-plugin-vue-inline-svg/compare/master...gionatan-lombardi:feature/svg-from-absolute-paths?expand=1#diff-168726dbe96b3ce427e7fedce31bb0bcR27) node method, from where the process has started.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,6 +18,17 @@ export default function (options) {
...
   return {
     name: 'vue-inline-svg',
+    resolveId: function (source) {
+      // Metches absolute paths
+      // of svgs in node_modules
+      // that are skipped by default in rollup
+      var regexp = /^(?!\.)\S+\.svg$/gi;
+      if (source.match(regexp)) {
+        var rootDir = process.cwd();
+        return { id: rootDir + '/node_modules/' + source, external: false };
```
Comment (e-e-e) [inline index.js:28]: Could you change this to use node's path module resolve?

Comment (gionatan-lombardi) [reply index.js]: Good point, thank you!

I've updated the PR.

Comment (gionatan-lombardi) [reply index.js]: Hi @e-e-e any news about this PR?

Comment (e-e-e) [reply index.js]: My apologies I will merge now.

**Comment by e-e-e** [comment]
I will publish to npm tomorrow hopefully.

File: index.js (hunk 1 above)
Comment (gionatan-lombardi) [reply index.js]: Thanks a lot!

**Pull request merged**